charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -46,7 +46,7 @@
           "$ref": "#/definitions/Protocol"
         }
```

Comment:
better to not checkin `taxonomy.json` as it is a dynamic file and its contents will depend on what is used to generate it (base + layers).

Reply:
I have made the file at par with the current version in master repository.

Reply:
Rebased the code.